pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go

```diff
@@ -76,6 +76,8 @@ type MonitorInfo struct {
 type RunnerInfo struct {
 	//Image of the runner pod
 	Image string `json:"image"`
```

Comment:
just thinking, would it be good to add omitempty to image as well?

Reply:
Yes!! Add the omitempty here also